catalyst/callbacks/metrics/scikit_learn.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Callable, Dict, Mapping, Union
+
+import sklearn
+import torch
+
+from catalyst.callbacks.metric import FunctionalBatchMetricCallback
+from catalyst.core.runner import IRunner
+from catalyst.metrics import FunctionalBatchMetric
+
+
+class SklearnCallback(FunctionalBatchMetricCallback):
+    """
+
+    Args:
+        keys:
+        metric_fn:
+        metric_key:
+        log_on_batch:
+    """
+
```

Comment:
could you please add docs to the metrics + to the official [API docs](https://github.com/catalyst-team/catalyst/blob/master/docs/api/metrics.rst)

Reply:
WIP🙂

Reply:
done